README.md

```diff
@@ -1 +1,15 @@
-base-sdks
+# Base SDK for Nitric
+
+Auto generated base-sdks which supports the Nitric framework. For more information, check out the main [Nitric repo](https://github.com/nitrictech/nitric).
+
+## Get in touch
+
+- Ask questions in [GitHub discussions](https://github.com/nitrictech/nitric/discussions)
+
+- Find us on [Twitter](https://twitter.com/nitric_io)
+
+- Send us an [email](maintainers@nitric.io)
```

Comment:
Should this be mailto:maintainers@nitric.io?

Reply:
Yep - will fix!